### ISSUE TITLE
Adding H2O to the Docker Official Images program

### ISSUE THREAD

**Issue by bardiharborow** [h2o/h2o#1977]
I'm just wondering if you would consider blessing/participating in a request to add H2O to the [Docker Official Images program](https://github.com/docker-library/official-images). If you have a moment, review of [the Dockerfile](https://github.com/bardiharborow/docker-h2o/blob/master/Dockerfile) would be appreciated.

**Comment by deweerdt** [comment]
@bardiharborow thank you for doing this, would you be maintaining the image? It looks good to me, albeit i haven't had the time to test it.

**Comment by bardiharborow** [comment]
@deweerdt yes I'm very prepared to take responsibility for that. I'm also happy to run this out of a private repository or move it to h2o/docker so you all can have access too, whatever you'd prefer.

**Comment by deweerdt** [comment]
I think the current location works. Let me know if i can help further.
